src/main/java/org/broadinstitute/hellbender/tools/dataflow/transforms/markduplicates/Utils.java

```diff
@@ -23,54 +23,11 @@
 import java.util.stream.StreamSupport;
 
 /**
- * The main transform for MarkDuplicates. Takes reads, returns reads with some reads marked as duplicates.
+ * Utility classes and functions for Mark Duplicates.
  */
-public class MarkDuplicates extends PTransform<PCollection<GATKRead>, PCollection<GATKRead>> {
-    private static final long serialVersionUID = 1l;
-
+final class Utils {
```

Comment:
Name this MarkDuplicatesUtils -- having lots of classes with the same/generic names is annoying in an IDE.